Library/Homebrew/test/cleanup_spec.rb

```diff
@@ -63,6 +63,11 @@
     end
   end
 
+  specify "::disk_cleanup_size" do
+    disk_cleanup_size = described_class.instance_variable_get(:@disk_cleanup_size)
+    expect(described_class.disk_cleanup_size).to eq(disk_cleanup_size)
+  end
```

Comment:
Instead of this test could just use `attr_reader :disk_cleanup_size` and replace the `def disk_cleanup_size` method.